UniversityofHelsinki/Styleguide
# Hero changes 7984

Removed overlay from hero-big and added box-decoration-break to hero title and ingress, image is also bigger

How to test:
1. confirm you are running node 8.4.0 or lower (styleguide does not support newer version)
2. run `npm install` and `gulp serve`
3. confirm that styleguide works and hero big is as described above.
